include/service/entities/article_service.js

```diff
@@ -132,11 +132,11 @@ module.exports = function ArticleServiceModule(pb) {
 
         //build out the ordering
         var order;
-        if (util.isArray(options.order)) {
+        if (util.isObject(options.order)) {
             order = options.order;
         }
         else {
-            order = [['publish_date', pb.DAO.DESC], ['created', pb.DAO.DESC]];
+            order = {'publish_date': pb.DAO.DESC, 'created': pb.DAO.DESC};
```

Comment:
Ok, part of the issue is mongo's documentation site.  When you click on the ```Cursor``` prototype and then click on the ```sort``` function in the left hand menu it takes you to the ```AggregationCursor``` page and not the generic cursor page.  Here is the correct link for the function:

http://mongodb.github.io/node-mongodb-native/2.0/api/Cursor.html#sort